packages/create-ship-app/src/scripts/full-stack.sh

```diff
@@ -22,12 +22,16 @@ function installService() {
   service="$1"
   service_dir="$2"
 
-  cp -a "ship/$service_dir/." "$service"
+  if [ "$service" == "deploy" ]; then
```

Comment:
let's create separate script for .net and koa

Reply:
created a separate script. 

creating a variable for path won't help because if we change the depth of the path we will need to change a lot of cd ../.. commands, and it's not consistent with current script